src/test/unit/views/status-description.test.ts

```diff
@@ -0,0 +1,53 @@
+import { expect } from 'chai';
+import request from 'supertest';
+
+import { app } from '../../../main/app';
+
+const PAGE_URL = '/status-description';
+
+let htmlRes: Document;
+
+describe('Status Description page', () => {
+  beforeAll(async () => {
+    await request(app).get(PAGE_URL).then(res => {
+      htmlRes = new DOMParser().parseFromString(res.text, 'text/html');
+    });
+  });
+
+  it('should display a back button with the correct value', () => {
+    const backLink = htmlRes.getElementsByClassName('govuk-back-link');
+    expect(backLink[0].innerHTML).contains('Back', 'Back button does not contain correct text');
+    expect(backLink[0].getAttribute('href')).equal('#', 'Back value does not contain correct link');
```

Comment:
this test is kind of redundant when you arent using href for navigation, can you change it to reflect the code in your actual back button?

Reply:
Actual back button is History(-1). It will be changed when actual page will be implemented. Currently, we do not have that page from which user will navigate to status description page. Once that page is implemented, back button will point to that one.